drivers/i2c/i2c_ll_stm32_v2.c

```diff
@@ -224,15 +224,23 @@ int i2c_stm32_target_register(const struct device *dev,
 
 	if (!data->slave_cfg) {
 		data->slave_cfg = config;
+		if (data->slave_cfg->flags == I2C_TARGET_FLAGS_ADDR_10_BITS)	{
+			LL_I2C_SetOwnAddress1(i2c, config->address, LL_I2C_OWNADDRESS1_10BIT);
+			LOG_DBG("i2c: slave #1 registered with 10-bit address");
```

Comment:
The verbage for i2c has changed from master/slave to controller/target, while I don't expect the entire driver to be reworked for this a log message should probably use the newer friendlier terminology.